sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLParserSuite.scala

```diff
@@ -861,7 +861,8 @@ class DDLParserSuite extends PlanTest with SharedSQLContext {
     assertUnsupported(sql2_view)
 
     val tableIdent = TableIdentifier("table_name", None)
-    val expected1_table = AlterTableDropPartitionCommand(
+
+    val expected1_table = AlterTableDropPartitionCommand.fromSpecs(
```

Comment:
Can you add tests case to check if the parser can accept the comparators added by this pr?

Reply:
sure, will do, thanks.